src/pages/landingpage/landingpage.css

```diff
@@ -0,0 +1,81 @@
+.landingpage-container{
+    min-height: 80vh;
+    max-height:100%;
+    display:flex;
+    align-items:center;
+    justify-content: space-around;
+    flex-wrap:wrap;
+    min-width:300px;
+
+    
+}
+.landingpage-img img{
+    flex-basis: 60%;
+    min-width:300px;
+}
+.landingpage-info{
+    flex-basis:40%;
+    align-items: center;
+    justify-content: center;
+}
+.Brand-name{
+    font-size:3rem;
+    font-family: cursive;
+    color:var(--primary-color);
+}
+.info-para{
+    font-size: 1.5rem;
+    margin-top: 2rem;
+}
+.go-to-notes-btn{
+    border:none;
+    padding:10px 10px;
+   max-width: fit-content;
+   font-size: 1.2rem;
+   margin-top:2rem;
+   display: flex;
+   align-items: center;
+   justify-content: center;
+   background-color: var(--primary-color);
+   color:white;
+   cursor:pointer;
```

Comment:
you have used color white Two times till now I think, so you can use variables for that

Reply:
will work on this aryan